File: fos-client/src/app/models/event-action.ts


    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
     

    export class EventAction    {
           
        public CanViewEvent: boolean = false;   
        public CanEditEvent: boolean = false;   
        public CanCloneEvent: boolean = false;   
        public CanCloseEvent: boolean = false;   
        public CanMakeOrder: boolean = false;   
        public CanSendRemind: boolean = false;   
        public CanViewOrder: boolean = false;   
        public CanViewEventSummary: boolean = false;
    }
         

File: fos-api/FOS/FOS.Model/Dto/EventAction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FOS.Model.Dto
{
    public class EventAction
    {
        public bool CanViewEvent { get; set; }
        public bool CanEditEvent { get; set; }
        public bool CanCloneEvent { get; set; }
        public bool CanCloseEvent { get; set; }
        public bool CanMakeOrder { get; set; }
        public bool CanSendRemind { get; set; }
        public bool CanViewOrder { get; set; }
        public bool CanViewEventSummary { get; set; }
    }
}
